raiden-ts/tests/unit/epics/monitor.spec.ts

```diff
@@ -110,124 +81,58 @@ describe('monitorRequestEpic', () => {
         // rate=WeiPerEther == 1:1 to SVT
         rateToSvt: { [token]: WeiPerEther as UInt<32> },
       }),
-      tokenMonitored({ token, tokenNetwork }),
-      channelOpen.success(
-        {
-          id: channelId,
-          settleTimeout,
-          isFirstParticipant,
-          token,
-          txHash,
-          txBlock: 121,
-          confirmed: true,
-        },
-        { tokenNetwork, partner },
-      ),
-      channelDeposit.success(
-        {
-          id: channelId,
-          participant: depsMock.address,
-          totalDeposit: bigNumberify(500) as UInt<32>,
-          txHash,
-          txBlock: 122,
-          confirmed: true,
-        },
-        { tokenNetwork, partner },
-      ),
-      channelDeposit.success(
+    );
+    await ensureChannelIsDeposited([raiden, partner], deposit);
+    await ensureChannelIsDeposited([partner, raiden], deposit);
+    await ensureTransferUnlocked([partner, raiden], amount);
+
+    await waitBlock(2);
```

Comment:
`ẁaitBlock(2)` doesn't wait 2 blocks, but emits the `blockNumber=2`, to all clients, which is wrong since it's smaller than the default starting block of 100. This just works here because it's not being used by the specific block, and instead only for waiting a block and the time for its event to be properly handled.
```suggestion
    await waitBlock();
```